modules/install-sno-about-single-node-openshift.adoc

```diff
@@ -0,0 +1,11 @@
+// This is included in the following assemblies:
+//
+// installing_bare_metal_single_node/install-sno-single-node-ocp-on-bare-metal.adoc
+
+[id="about-single-node-openshift_{context}"]
```

Comment:
Match the ID to the file name as described in the second item under [Module file metadata](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata) in the guidelines.